docs/OBOOpsLogistics.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: doc
+id: OBOOpsLogistics
```

Comment:
Didnt @cthoyt advice us that this was redundant?

Reply:
yes, he did. however, it's my best guess that it is needed for the html (which is currently missing) to be created. we'll see.

Reply:
You can already go to https://obofoundry.org/docs/OBOOpsLogistics.html on the live site. The ID is strictly not relevant for anything. The only issue was the typo in the table of contents.

Reply:
hmm, ok, i thought i'd tried going there and it didn't work. if that url works, then you're right, the id is not needed. i will close this PR.

Reply:
do i even have power to close this PR or does someone else need to do it?

Reply:
actually, the other file edit in this same PR is the one we want to keep. is there a way to just remove this edit from the PR? (sorry for my gh ignorance)

Reply:
```suggestion
```